Sourcery/Parsing/FileParser.swift

```diff
@@ -533,11 +533,14 @@ extension FileParser {
 
     internal func parseParameter(_ source: [String: SourceKitRepresentable]) -> MethodParameter? {
         guard let (name, _, _) = parseTypeRequirements(source),
-            let type = source[SwiftDocKey.typeName.rawValue] as? String else { return nil }
+              let type = source[SwiftDocKey.typeName.rawValue] as? String else {
+            return nil
+        }
 
+        let inOut = type.hasPrefix("inout ")
```

Comment:
I have the same doubts about where it belongs, I think if it would be treated not as a special kind of _type_, the same as `throws`, it would be an attribute, maybe. I'm fine with keeping it `MethodParameter`, but we still need to trim it in `unwrappedTypeName` of `TypeName`.